x-pack/metricbeat/module/prometheus/remote_write/data.go

```diff
@@ -235,10 +220,11 @@ func (g *remoteWriteTypedGenerator) processPromHistograms(eventList map[string]m
 
 		e := eventList[labelsHash]
 
-		hist := dto.Histogram{
+		hist := p.Histogram{
 			Bucket: histogram.buckets,
 		}
 		name := strings.TrimSuffix(histogram.metricName, "_bucket")
+		_ = name // skip noisy linter
```

Comment:
is it about golangci-linter? which warning/error does it fix?

Reply:
not sure what the issue here, but in some cases linter fails on darwin only - sample PR https://github.com/elastic/beats/pull/34175#issuecomment-1375611992, if it is your case - might be better just ignore it, instead of adding this line

Reply:
The issue is that the linter complains because it thinks that the variable is never used. This is not true but make the whole CI execution to fail.

Reply:
Ah good to know. Let me try to remove it and check what happens.